provisioners/test-ldap.yml

```diff
@@ -0,0 +1,22 @@
+---
+- hosts: all
+
+  tasks:
+    
+    - name: ldap
+      pip:
+        name=python-ldap
+      sudo: yes
+
+    - name: hit ldap with script
+      shell: "echo check url is passed $ldap_server"
+         # shell: "echo check url is passed $ldap_server"
+         # python ckanext-cfpb-authtesting/ldap_auth.py
+      environment:
+        # need to ensure that this is the ldap plugin and [1] is a bad way to do this
+        # ckanext.*.* is wrong... [1].config is a multi-line string!
+        ldap_server: "{{ ckan_custom_plugins[1].config.ckanext.ldap.uri }}" 
```

Comment:
you may be able to use a jinja2 filter on this:
```
ckan_custom_plugins | selectattr("name", "ldap").whatever.foo.bar
```
http://jinja.pocoo.org/docs/dev/templates/#selectattr